src/main/java/at/porscheinformatik/sonarqube/licensecheck/gradle/GradleDependencyScanner.java

```diff
@@ -28,6 +28,9 @@
 
 public class GradleDependencyScanner implements Scanner
 {
+    public static final String JSON_REPORT_PATH_PROPERTY = "sonar.licenseCheck.jsonReportPath";
+    public static final String JSON_REPORT_PATH_DEFAULT = "build/reports/dependency-license/license-details.json";
```

Comment:
Please use `"build" + File.separator + "reports" + File.separator + "dependency-license" + File.separator + "license-details.json"` as the default value